dev/com.ibm.ws.channelfw/bnd.bnd

```diff
@@ -81,7 +84,8 @@ instrument.disabled: true
 	com.ibm.ws.timer;version=latest,\
 	com.ibm.ws.kernel.security.thread,\
 	com.ibm.ws.kernel.boot.core;version=latest, \
-	com.ibm.ws.org.osgi.annotation.versioning;version=latest
+	com.ibm.ws.org.osgi.annotation.versioning;version=latest, \
+	com.ibm.ws.wsbytebuffer
```

Comment:
When to include ';version=latest' ?

Reply:
As wsbytebuffer is internal, I do not expect more than one version to exist at one time.